### DISCUSSION TITLE
problem with the new argon skin

### DISCUSSION THREAD

**Discussion by Nepp0** [ppy/osu#20446] [Q&A] [answered]
sliders can block hitcircle visibility making them harder to read if they overlap.
example:
![Screenshot from 2022-09-24 11-44-30](https://user-images.githubusercontent.com/98234215/192088055-e4368148-ee7a-473f-8f90-5e4f372c3df2.png)
compared to triangles skin:
![Screenshot from 2022-09-24 11-44-43](https://user-images.githubusercontent.com/98234215/192088069-ed30d496-6716-475f-a0de-2d0466532fb1.png)

**Comment by peppy** [comment, marked as answer]
Give it a while and it might become easier to follow.

We'll add customisation of slider track opacity in the future, but the default will be 100%.
